webpack.config.js

```diff
@@ -279,8 +290,6 @@ const settingsPage = {
 			requestToHandle: ( handle ) => {
 				switch ( handle ) {
 					case '@wordpress/api-fetch':
-					case '@wordpress/dom-ready':
-					case '@wordpress/html-entities':
```

Comment:
We were loading these from core because they are relatively stable since 5.0, but they're also small, so bundling them in our JS is just a drop in the bucket.